scripts/upgrade

```diff
@@ -56,6 +56,10 @@ ynh_script_progression --message="Backing up the app before upgrading (may take
 # Backup the current version of the app
 ynh_backup_before_upgrade
 ynh_clean_setup () {
+  # Remove the PostgreSQL db in case of error during the migration.
+  if [ -n "$remove_psql_in_case_of_error" ]; then
+    ynh_psql_remove_db --db_user=$db_user --db_name=$db_name
+  fi
```

Comment:
Yay, the Postgres DB not being deleted caused me a lot of hard-to-debug issues, thanks for this!